crates/daphne/src/vdaf/mod.rs

```diff
@@ -277,7 +277,7 @@ pub enum VdafPrepMessage {
     #[cfg(feature = "experimental")]
     MasticShare(Field64),
     #[cfg(feature = "experimental")]
-    Pine128Share(crate::pine::msg::PrepShare<Field128, 16>),
+    Pined64HmacSha256Aes128(crate::pine::msg::PrepShare<Field64, 32>),
```

Comment:
32 is the seed size? out of curiosity, why a smaller field uses a larger seed size?

Reply:
The two are unrelated:

* Pine128 uses XofTurboShaek128, which at the moment has a 16-byte seed
* Pine64HmacSha256Aes128 uses XofHmacSha256Aes128 (a non-standard XOF implemented in this repo), which has a 32 bytes eed.